tardis/adapters/sites/fakesite.py

```diff
@@ -64,15 +64,7 @@ def get_resource_boot_time(self, resource_attributes: AttributeDict) -> float:
 
     @property
     def machine_meta_data(self) -> AttributeDict:
-        return self.configuration.MachineMetaData[self._machine_type]
-
-    @property
-    def machine_type(self) -> str:
-        return self._machine_type
-
-    @property
-    def site_name(self) -> str:
-        return self._site_name
+        return self._configuration.MachineMetaData[self._machine_type]
```

Comment:
From my point of view, this can be dropped completely.

Reply:
Good catch! I will remove it.